build.gradle

```diff
@@ -350,7 +350,8 @@ coreProjects.each { coreProject ->
     def assembleTasks = collectTasksRecursively(coreProject, 'assemble')
     def publishTasks = collectTasksRecursively(coreProject, 'publish')
 
-    // Add these tasks as dependencies of the release task
-    release.dependsOn assembleTasks
-    release.dependsOn publishTasks
+    // Explicitly declare release task for better gradle compatibility
+    def releaseTask = tasks.named('release').get()
```

Comment:
I encountered issues in with this build file in my environment:

```
Build file '/path/to/data-prepper/build.gradle' line: 354

A problem occurred evaluating root project 'opensearch-data-prepper'.
> No signature of method: java.lang.String.dependsOn() is applicable for argument types: (ArrayList) values: [[task ':data-prepper-api:assemble']]
```

Appears my gradle did not see "release" as a task but as a String. This workaround will make sure to initialize it as a task.

Reply:
Yes, this makes sense.